### PR TITLE
My attempt to make homepage responsive

### PR DESCRIPTION
**Pull request by daigofuji**
Addresses #10 
Not familiar with the mixins used so steer me to right direction if I am doing it wrong. Will comment my changes below...

### CODE REVIEW

File: source/_assets/stylesheets/_index.scss [hunk 1]
```diff
@@ -129,8 +140,12 @@
   @include transition(1s);
   @include display(flex);
   @include align-items(flex-end);
-  padding: 6rem;
+  padding: 2rem;
+  @include media(em(960)) {
+    padding: 6rem;
+  }
   width: 50%;
+  height: 100vh;
```
Comment (daigofuji) [inline source/_assets/stylesheets/_index.scss]: without this, on firefox, you can't see Yogi's face.

File: source/_assets/stylesheets/_index.scss [hunk 2]
```diff
@@ -42,8 +42,10 @@
 
   .logo {
     width: 125px;
-    margin: 0 auto 3rem;
-
+    margin: 0 auto 1rem;
+    @include media(em(520)) {
```
Comment (daigofuji) [inline source/_assets/stylesheets/_index.scss]: Am I using media mixin right here?

**Comment by daigofuji** [comment]
@kylefiedler what do you think?

File: source/_assets/stylesheets/base/_lists.scss [hunk 3]
```diff
@@ -3,6 +3,7 @@ ol {
   margin: 0;
   padding: 0;
   list-style-type: none;
+  overflow: auto; //clearfix
```
Comment (daigofuji) [inline source/_assets/stylesheets/base/_lists.scss]: on mobile, floated lis on prizes and judges are all bunched up. this fixes it but should I have not touched /base/*?

**Comment by daigofuji** [comment]
Should I ignore everything but /source? A lot of noise in those commits.

File: source/_assets/stylesheets/base/extends/_button.scss [hunk 4]
```diff
@@ -30,7 +30,10 @@
   font-size: em(24);
   letter-spacing: 1px;
   line-height: 1;
-  padding: 0.4em 1.5em;
+  padding: 0.4em 0.75em;
+  @include media(em(520)) {
```
Comment (kylefiedler) [inline source/_assets/stylesheets/base/extends/_button.scss]: Line above

File: source/_assets/stylesheets/base/extends/_button.scss [hunk 5]
```diff
@@ -30,7 +30,10 @@
...
+    padding: 0.4em 1.5em;
+  }
   text-transform: uppercase;
```
Comment (kylefiedler) [inline source/_assets/stylesheets/base/extends/_button.scss]: Put this above the media query

File: source/_assets/stylesheets/_index.scss [hunk 6]
```diff
@@ -140,19 +155,30 @@
...
+    }
     -webkit-font-smoothing: antialiased;
     color: $light-tan;
     font-family: $sub-header-font-family;
-    font-size: em(26);
     line-height: 1.3;
-    margin-bottom: 3rem;
+    font-size: em(24);
+    margin-bottom: 1rem;
+    @include media(em(520)) {
```
Comment (kylefiedler) [inline source/_assets/stylesheets/_index.scss]: Empty line above

File: source/_assets/stylesheets/_index.scss [hunk 7]
```diff
@@ -140,19 +155,30 @@
...
+      font-size: em(40);
+    }
     &::after {
       background-color: shade($tan, 80);
     }
   }
 
   p {
+    display: none;
+    @include media(em(360)) {
```
Comment (kylefiedler) [inline source/_assets/stylesheets/_index.scss]: Can you put the media query below all the other styles for `<p>`

File: source/_assets/stylesheets/_index.scss [hunk 8]
```diff
@@ -140,19 +155,30 @@
 
   h2 {
     color: #fff;
-
+    font-size: em(26);
+    @include media(em(520)) {
```
Comment (kylefiedler) [inline source/_assets/stylesheets/_index.scss]: Empty line above and below the media query

File: source/_assets/stylesheets/_index.scss (hunk 1 above)
Comment (kylefiedler) [inline source/_assets/stylesheets/_index.scss]: :ok_hand: Want to use bourbon's `size` mixin?

File: source/_assets/stylesheets/_index.scss [hunk 9]
```diff
@@ -55,14 +57,23 @@
...
+  } 
   pointer-events: none;
   text-align: center;
   width: 100%;
 
   h2 {
     color: #fff;
-    font-size: em(52);
+    font-size: em(36);
+    @include media(em(520)) {
```
Comment (kylefiedler) [inline source/_assets/stylesheets/_index.scss]: Move the media query below the default `h2` styles

File: source/_assets/stylesheets/_index.scss [hunk 10]
```diff
@@ -55,14 +57,23 @@
 }
 
 .hero {
-  @include position(absolute, 20vh null null null);
+  @include position(absolute, 3vh null null null);
+  @include media(em(960)) {
```
Comment (kylefiedler) [inline source/_assets/stylesheets/_index.scss]: move media queries below default styles for `.hero`

File: source/_assets/stylesheets/_index.scss (hunk 2 above)
Comment (kylefiedler) [inline source/_assets/stylesheets/_index.scss]: :+1:

Comment (kylefiedler) [inline source/_assets/stylesheets/_index.scss]: Line above

File: source/_assets/stylesheets/_index.scss [hunk 11]
```diff
@@ -42,8 +42,10 @@
 
   .logo {
     width: 125px;
-    margin: 0 auto 3rem;
-
+    margin: 0 auto 1rem;
+    @include media(em(520)) {
+      margin: 0 auto 3rem;
+    }
```
Comment (kylefiedler) [inline source/_assets/stylesheets/_index.scss]: Empty line below

**Comment by kylefiedler** [comment]
@daigofuji can you attach a screen shot? Code looks good just a couple formatting things. I keep to the thoughtbot Sass styleguide: https://github.com/thoughtbot/guides/tree/master/style/sass

And yea you can ignore whats not in `/source` the rest is the Jeckyll build.

**Comment by daigofuji** [comment]
Cool will make those changes/push and attach screenshot. Thanks for reviewing.

**Comment by daigofuji** [comment]
For landscape phone, I had to do kind of ugly media queries. If you have better approach, let me know...
screen shot 2015-02-20 at 4 09 56 pm
screen shot 2015-02-20 at 4 10 06 pm
screen shot 2015-02-20 at 4 10 24 pm
screen shot 2015-02-20 at 4 10 32 pm

**Comment by daigofuji** [comment]
I have also removed build folder from this repo. Let me know if that cause any problems...

**Comment by kylefiedler** [comment]
:+1:  looks good

**Comment by daigofuji** [comment]
Merging! But does it get automatically applied to master or do I have to run a script?

**Pull request merged**